.github/workflows/main.yaml

```diff
@@ -75,83 +70,45 @@ jobs:
           sudo apt-get install --reinstall g++-8
-      - name: Set up musl on Linux
-        if: matrix.build == 'linux-musl-x64'
-        run: |
-          sudo apt-get update -y
-          sudo apt-get install -y musl-dev musl-tools
       - name: Install Rust ${{ matrix.rust }}
         uses: actions-rs/toolchain@v1
         with:
           toolchain: ${{ matrix.rust }}
           target: ${{ matrix.target }}
           override: true
-      #- name: Cache
-      #  uses: actions/cache@master
-      #  with:
-      #    # Note: crates from the git repo always get rebuilt
-      #    # so we cache only those subdirectories of target/{debug|release} that
-      #    # contain the build output for crates that come from the registry.
-      #    path: |-
-      #      .cargo_home
-      #      target/*/.*
-      #      target/*/build
-      #      target/*/deps
-      #    key: ${{ matrix.os }}-${{ matrix.rust }}-${{ hashFiles('Cargo.lock') }}
-      #    restore-keys: |
-      #      ${{ matrix.os }}-${{ matrix.rust }}-${{ hashFiles('Cargo.lock') }}
-      # # Install sccache
-      # - uses: actions/cache@master
-      #   with:
-      #     path: ${{ runner.tool_cache }}/cargo-sccache
-      #     key: cargo-sccache-bin-${{ env.CARGO_SCCACHE_VERSION }}
-      # - name: Install sccache
-      #   run: |
-      #     echo "${{ runner.tool_cache }}/cargo-sccache/bin" >> $GITHUB_PATH
-      #     cargo install sccache --version ${{ env.CARGO_SCCACHE_VERSION }} --root ${{ runner.tool_cache }}/cargo-sccache
-      # - name: Start sccache
-      #   run: |
-      #     ${{ runner.tool_cache }}/cargo-sccache/bin/sccache --start-server
-      #     ${{ runner.tool_cache }}/cargo-sccache/bin/sscache -s
-      #     echo "RUSTC_WRAPPER=${{ runner.tool_cache }}/cargo-sccache/bin/sccache" >> $GITHUB_ENV
       - name: Install LLVM (Windows)
-        if: matrix.os == 'windows-latest'
-        run: choco install llvm
-        # run: |
-        #   curl --proto '=https' --tlsv1.2 -sSf https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/LLVM-11.0.1-win64.exe -L -o llvm-installer.exe
-        #   7z x llvm-installer.exe -oC:/llvm-11
-        #   echo C:/llvm-11/bin >> $GITHUB_PATH
-        #   echo "LLVM_SYS_110_PREFIX=C:/llvm-11" >> $GITHUB_ENV
-        #   echo "LIBCLANG_PATH=C:/llvm-11/bin/libclang.dll" >> $GITHUB_ENV
-      - name: Install LLVM (Unix)
-        if: matrix.os != 'windows-latest' && matrix.build != 'linux-musl-x64'
+        if: matrix.os == 'windows-latest' && !matrix.llvm_url
+        shell: cmd
         run: |
-          curl --proto '=https' --tlsv1.2 -sSf ${{ matrix.llvm_url }} -L -o llvm.tar.xz
-          mkdir -p ${{ env.LLVM_DIR }}
-          tar xf llvm.tar.xz --strip-components=1 -C ${{ env.LLVM_DIR }}
-          echo "${{ env.LLVM_DIR }}/bin" >> $GITHUB_PATH
-          echo "LLVM_SYS_110_PREFIX=${{ env.LLVM_DIR }}" >> $GITHUB_ENV
+          choco install llvm
```

Comment:
Does it make sense to keep installing llvm on windows? We don't support llvm on windows yet, right?

Reply:
fair point: #14